src/gui/clips/ClipView.cpp

```diff
@@ -425,7 +425,7 @@ void ClipView::setColor(const QColor* color)
 void ClipView::dragEnterEvent( QDragEnterEvent * dee )
 {
 	TrackContentWidget * tcw = getTrackView()->getTrackContentWidget();
-	TimePos clipPos = TimePos( m_clip->startPosition() );
+	auto clipPos = TimePos(m_clip->startPosition());
```

Comment:
Maybe just `TimePos clipPos(m_clip->startPosition());`?

Reply:
Sure, but I'll use ``{}``, rather than ``()``, as per https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax. I think we should start using this kind of initialization, given that we would want to adhere to the guidelines, and is just a better to initialize in general. I wouldn't consider this a hard requirement by any means however, but just something we should prefer when given the opportunity.